src/Item/access.ts

```diff
@@ -27,9 +29,17 @@ export async function hasAccess(
     isCommitteeMember(eth_address),
   ])
 
-  const isManager: boolean = collection
-    ? isCollectionManager(eth_address, collection)
-    : false
+  let isManager = false
+  if (collection) {
+    if (isTPCollection(collection)) {
+      isManager = await thirdPartyAPI.isManager(
+        collection.third_party_id,
+        eth_address
+      )
+    } else {
+      isManager = isCollectionManager(eth_address, collection)
```

Comment:
What if we add a test to verify that has access works as intended with this change?

Reply:
Same as [this comment](https://github.com/decentraland/builder-server/pull/382#discussion_r798929263) although seeing this here makes me want to create both spec files and I think I'll do that, I still want to know your opinion